modules/deploy/pages/deployment-option/self-hosted/manual/sizing.adoc

```diff
@@ -105,7 +100,7 @@ size=10G
 ioengine=libaio
 iodepth=16
 EOF
-$ sudo fio fio-seq-write.job
+sudo fio fio-seq-write.job
```

Comment:
Looks good but I'm also wondering if we do this elsewhere in our docs where we have shell code snippets?

Reply:
I ran a search and I think the rest are fine; e.g., see [here](https://docs.redpanda.com/current/deploy/deployment-option/self-hosted/manual/production/production-deployment/) or [here](https://docs.redpanda.com/current/deploy/deployment-option/self-hosted/manual/production/production-readiness/).